docs/ossm/ossm2-migration/README.md

```diff
@@ -127,3 +127,8 @@ oc get smcp <smcp-name> -n <smcp-namespace> -o jsonpath='{.spec.mode}'
 ```
 
 For `MultiTenant` meshes, follow [these instructions](./multi-tenancy/README.md). For `ClusterWide` meshes, follow [these instructions](./cluster-wide/README.md). When the migration is finished, follow [these instructions](./cleaning-2.6/README.md) to remove OpenShift Service Mesh 2.6.
+
+## Post-migration Checklist
+
+After the migration is done, you can visually check whether all data plane namespaces are migrated to the newer version of OSSM by checking it in Kiali Mesh page.
```

Comment:
Maybe just add "optionally", because not users must be using kiali addon

Reply:
thanks, have rephased